From fd7460baf75c48617aa2e920dc86ac2726b8a36f Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Mon, 14 May 2018 11:06:35 +0800
Subject: [PATCH 3785/5242] MLK-18299 clk: imx8mm: add 594mhz for video pll on
 imx8mm

commit  f53ebc391b6d6836b4625323f48b8999dd0f103a from
https://source.codeaurora.org/external/imx/linux-imx.git

Add 594MHz config support for video pll on imx8mm. lcdif
driver need this frequency setpoint.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Reviewed-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-imx8mm.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/imx/clk-imx8mm.c b/drivers/clk/imx/clk-imx8mm.c
index 379a647..326b1c5 100644
--- a/drivers/clk/imx/clk-imx8mm.c
+++ b/drivers/clk/imx/clk-imx8mm.c
@@ -81,6 +81,7 @@ enum {
 
 static const struct imx_int_pll_rate_table imx8mm_videopll_tbl[] = {
 	PLL_1443X_RATE(650000000U, 325, 3, 2, 0),
+	PLL_1443X_RATE(594000000U, 198, 2, 2, 0),
 };
 
 static const struct imx_int_pll_rate_table imx8mm_drampll_tbl[] = {
-- 
1.7.9.5

